Xamarin.Forms.Platform.Android/Renderers/ImageRenderer.cs

```diff
@@ -120,5 +124,25 @@ public override bool OnTouchEvent(MotionEvent e)
 
 			return _motionEventHelper.HandleMotionEvent(Parent, e);
 		}
+
+		void SetTint()
+		{
+			if (Control == null || Element == null)
+				return;
+
+			if (Element.TintColor.Equals(Xamarin.Forms.Color.Transparent))
```

Comment:
The condition is not different between Android and iOS renderer. Keep consistency. 
```
if( Element.TintColor == Color.Transparent)
{
}
else
{
}
```